src/ert/run_models/iterated_ensemble_smoother.py

```diff
@@ -72,7 +72,7 @@ def _runAndPostProcess(
     ) -> None:
         phase_msg = (
             f"Running iteration {run_context.iteration} of "
-            f"{self.phaseCount() - 1} simulation iterations..."
+            f"{self.phaseCount() - 1} experiment iterations..."
```

Comment:
Think we can just simplify the sentence all together here
```suggestion
            f"{self.phaseCount() - 1}..."
```